helpers.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+class Helper {
```

Comment:
I wonder if this _really_ needs to be a class. There are no shared methods and `urlHash()` doesn't even need any of the object properties. I think I'd consider one of two approaches:

1. Just use an object literal, so that we don't have to instantiate the thing.
2. Just have a bunch of lightweight functions that you require as needed.

Also, if you just use `/` as the base of the URL, the browser will infer the protocol and domain. Sure, that's nitpicky, but it pretty much eliminates the need for `urlGen` and `liveUrlGen`. (You would then however, have to figure out how to do this on the client, which is not particularly hard but something to be done none-the-less.)